w3af/core/data/context/context/javascript.py

```diff
@@ -23,12 +23,16 @@
 
 from w3af.core.data.context.context.base import BaseContext
 from w3af.core.data.context.constants import CONTEXT_DETECTOR
+from w3af.core.data.context.utils import encode_payloads, decode_payloads
 
 STRING_DELIMITERS = {'"', "'"}
+# According to http://www.ecma-international.org/ecma-262/6.0/index.html
+#                                                   #sec-line-terminators
+LINE_TERMINATORS = {'\n', '\r', u'\u2028', u'\u2029'}
 
 
 class ScriptSingleLineComment(BaseContext):
-    CAN_BREAK = {'\n', '\r'}
+    CAN_BREAK = LINE_TERMINATORS
```

Comment:
It is not necessary due to changes in the PR case with `\u2028` and `\u2029` can leave only for Reflected-XSS and doesn't send it each time.
But if you think we need to check this case for persistent XSS - I'll add it.